### ISSUE TITLE
Can't enter new hotkeys in config menu because no way to exit hotkey configuring AND accept keys pressed

### ISSUE THREAD

**Issue by e-motiv** [blackd/Inventory-Profiles#111]
The problem is there is no key to "accept" or "finalize" the hotkey, once you are configuring it.
Trace:

1. I enter config => Hotkeys
2. I got to any hotkey and click on it
3. Now I am inside it an whatever I hit on the keyboard it registers it, but also "enter" the key to say "OK, accept this combination now"
4. So I hit my hotkey, say "ALT + r" , but when I hit enter to accept this, the key shows "ALT + r + Enter". I cannot get out of it except by pressing ESC, which cancels it.
5. (Clicking the mouse button als wrongle sets the key to "Left Button"

In other words, there is no way to configure a hotkey, except making it a mouse button, since only the mouse button and escape exits the hotkey config mode and both don't end up with your keys pressed.

Java Edition 1.18.1
Forge 39.0.79

**Comment by blackd** [comment]
do you have better f3 installed also?

**Comment by blackd** [comment]
also which version of the mod

**Comment by e-motiv** [comment]
Thank you! Confirm fixed!
